src/pal/pal_openbsd.h

```diff
@@ -0,0 +1,95 @@
+#pragma once
+
+#if defined(__OpenBSD__) && !defined(_KERNEL)
+#  include "../ds/bits.h"
+#  include "../mem/allocconfig.h"
+
+#  include <stdio.h>
+#  include <string.h>
+#  include <sys/mman.h>
+
+namespace snmalloc
+{
+  class PALOBSD
+  {
+  public:
+    /**
+     * Bitmap of PalFeatures flags indicating the optional features that this
+     * PAL supports.
+     */
+    static constexpr uint64_t pal_features = AlignedAllocation | LazyCommit;
```

Comment:
It was

> There's a lot of code copied and pasted from the FreeBSD PAL here. Please can you pull that out into a common generic BSD PAL? I think that the OpenBSD one will likely be the generic one and FreeBSD is that plus the addition of support for aligned allocation support.

It was to avoid too many `ifdef`

Reply:
> If you can't guarantee alignment, then don't advertise that you support aligned allocation - the code using the PAL will fix this for you.

Yes useless flag indeed.

Reply:
I meant pull out the common functions into a shared superclass, not add a mess of `ifdef`s...

Reply:
Sorry my bad :-)